src/configuration.cc

```diff
@@ -16,10 +16,12 @@
 
 #include "configuration.h"
 
+#include <boost/algorithm/string.hpp>
 #include <boost/program_options.hpp>
 #include <iostream>
 #include <map>
 #include <fstream>
```

Comment:
Let's alphabetize these while we're at it.

Reply:
done